packages/messages/src/browser/notification-component.tsx

```diff
@@ -74,14 +75,14 @@ export class NotificationComponent extends React.Component<NotificationComponent
                 <div className='theia-notification-list-item-content-main'>
                     <div className={`theia-notification-icon theia-notification-icon-${type}`} />
                     <div className='theia-notification-message'>
-                        <span dangerouslySetInnerHTML={{ __html: message }} onClick={this.onMessageClick} />
+                        <span dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(message) }} onClick={this.onMessageClick} />
```

Comment:
Since we explicitly handle the `sanitize`, we can suppress the warning we get:

```suggestion
                        <span
                            // eslint-disable-next-line react/no-danger
                            dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(message) }}
                            onClick={this.onMessageClick}
                        />
```

Reply:
Done :-)